src/Acquisition.jl

```diff
@@ -29,6 +30,25 @@ struct AcquisitionResults{S<:AbstractGNSS,T}
     }
 end
 
+function Base.show(io::IO, ::MIME"text/plain", acq_channels::Vector{Acquisition.AcquisitionResults{T1,T2}}) where {T1,T2}
+    header = ["PRN"; "CN0"; "Carrier doppler (Hz)"; "Code phase (chips)"]
+    data = Matrix{Any}(undef, length(acq_channels),length(header))
+
+    for (idx,acq) in enumerate(acq_channels)
+        data[idx,1] = acq.prn
+        data[idx,2] = acq.CN0
+        data[idx,3] = acq.carrier_doppler
+        data[idx,4] = acq.code_phase
+    end
```

Comment:
This can be reduced to
```suggestion
    data = reduce(vcat, map(acq -> [acq.prn, acq.CN0, acq.carrier_doppler, acq.code_phase]', acq_channels))
```